ote/resources/public/language/en.edn

```diff
@@ -225,7 +225,8 @@
 
   :brokerage? "Jos kyseessä on Liikenteen turvallisuusvirasto Trafille ilmoitettava välityspalvelu, laita alle raksi."
 
-  :operation-area "Add the service operation area (for example province) by typing the name in the text field and selecting it from the list. You can also draw the area with the map drawing tools. You can name the area freely (like \"Itäkeskus taxi station\")."
+  :operation-area "Lisää toiminta-alue kirjoittamalla alueen tai kohteen nimi \"Hae paikka nimellä\" -tekstikenttään ja valitsemalla haluttu alue avautuvalta listalta (tällä hetkellä haettavissa maakunta, kunta, postinumero, Suomi). Voit samalla tavalla lisätä useita alueita.
```

Comment:
Why replace English translated text with Finnish? Translate message. (same for all messages in en.edn file)

Reply:
Koska teksti muuttuu ja meillä ei taida olla vielä missään virallista englantia?
Otanko kaikki muutokset pois tästä tiedostosta?